server/utilities/trimfunction.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable eol-last */
+const convertName = name => (name.charAt(0).toUpperCase() + name.slice(1)).trim();
+
+export default convertName;
```

Comment:
This file name and method name should correlate. Make up on your mind what this does and name them accordingly

Reply:
Ok, noted